README.md

```diff
@@ -1,5 +1,28 @@
 # Deployment Guide
 
+The retail industry is modernizing their applications, many of which are latency sensitive, need high availability and would need distributed edge computing capabilities in order to support them. Azure Edge Zones combined with Couchbase’ highly scalable, distributed and embedded database can provide the perfect platform for such applications.
```

Comment:
Can we not mention the retail industry use case from the introduction? From the discussions that we had, it would be best to keep it generic. How about something like:

Couchbase's highly scalable and distributed database running on Azure public MEC provides the perfect platform to deploy latency sensitive applications at the operator's edge. Use cases such as next-generation retail, gaming, manufacturing can leverage this solution to deploy their mission critical applications.